src/Backoffice/Task/App/Requests/UpsertTaskRequest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Task\App\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rule;
+use Lightit\Backoffice\Employee\Domain\Models\Employee;
+use Lightit\Backoffice\Task\Domain\DataTransferObjects\TaskDto;
+use Lightit\Backoffice\Task\Domain\Enums\TaskStatus;
+use Lightit\Backoffice\Task\Domain\Models\Task;
+
+class UpsertTaskRequest extends FormRequest
+{
+    public const TITLE = 'title';
+
+    public const DESCRIPTION = 'description';
+
+    public const STATUS = 'status';
+
+    public const EMPLOYEE_ID = 'employee_id';
+
+    public const TASK_ID = 'id';
+
+    /**
+     * @return array<string, mixed>
+     */
+    public function rules(): array
+    {
+        return [
+            self::TITLE => ['required', 'string', 'min:2', 'max:80'],
+            self::DESCRIPTION => ['required', 'string', 'min:2'],
+            self::STATUS => ['required', Rule::enum(TaskStatus::class)],
+            self::EMPLOYEE_ID => ['required', Rule::exists(Employee::class, 'id')],
+            self::TASK_ID => ['sometimes', Rule::exists(Task::class, 'id')],
+        ];
+    }
+
+    public function toDto(): TaskDto
+    {
+        return new TaskDto(
+            title: $this->string(self::TITLE)->toString(),
+            description: $this->string(self::DESCRIPTION)->toString(),
+            status: $this->string(self::STATUS)->toString(),
```

Comment:
Regarding the comment in the Task model, as you have an Enum for the Task statuses, this could be, instead:

```suggestion
            status: $this->enum(self::STATUS, TaskStatus::class),
```

This way, you will be dealing with an Enum instance instead of a random string